requirements.txt

```diff
@@ -1,6 +1,6 @@
 distribute
 argparse
 sqlalchemy>=0.7
-IPython>=0.10
+IPython>=0.10,<4.0.0b1
```

Comment:
doesn't work. <4, <4.0, <4.0.0 don't work because technically 4.0.0b1 is less than all of those i guess :(